src/app/pages/sharing/components/shares-dashboard/smb-card/smb-card.component.spec.ts

```diff
@@ -160,27 +159,12 @@ describe('SmbCardComponent', () => {
   });
 
   it('handles edit Share ACL', async () => {
-    const deleteIcon = await table.getHarnessInCell(IxIconHarness.with({ name: 'share' }), 1, 4);
-    await deleteIcon.click();
-
-    expect(spectator.inject(WebSocketService).call).toHaveBeenCalledWith(
-      'pool.dataset.path_in_locked_datasets',
-      ['/mnt/APPS/smb1'],
-    );
+    const editIcon = await table.getHarnessInCell(IxIconHarness.with({ name: 'share' }), 1, 4);
+    await editIcon.click();
 
     expect(spectator.inject(WebSocketService).call).toHaveBeenCalledWith(
       'sharing.smb.getacl',
       [{ share_name: 'homes' }],
     );
   });
```

Comment:
Would be good to fix those tests properly, i.e. check that user was redirected to a correct page or correct slide in was opened.